INSTALL.md

```diff
@@ -274,23 +274,17 @@ Install Homebrew
 Install required packages:
 
 ```
-brew install cmake cspice fmt gettext libpng lua qt5 jpeg eigen
+brew install cmake glew fmt gettext libpng lua qt5 jpeg eigen
```

Comment:
why did you remove cspice?

Reply:
i think it is optional, i'll list it in optional later

Reply:
done